docs/docusaurus/docs/cloud/alerts/manage_alerts.md

```diff
@@ -8,11 +8,13 @@ Stay informed about the health of the Data Assets that you care about by enablin
 
 ## Email alert default settings
 
-By default, alerts are enabled for every Data Asset you create and disabled for all other Data Assets. You can change these settings at any time.
+Email alerts are configured on a per-user basis; your email alert settings do not affect the email alerts of other users in your organization.
+
+By default, alerts are enabled for you for every Data Asset you create and disabled for all Data Assets created by others. You can change these settings at any time.
```

Comment:
**note for reviewers:** I reorganized and slightly reworded this because on my initial read I was confused by the old first paragraph until I read the third